modules/virt-installing-virtctl-client.adoc

```diff
@@ -1,66 +1,75 @@
 // Module included in the following assemblies:
 //
-// virt/install/virt-enabling-virtctl.adoc
+// * virt/install/virt-installing-virtctl.adoc
 
 :_content-type: PROCEDURE
 [id="virt-installing-virtctl-client_{context}"]
-= Installing the virtctl client
+= Installing the virtctl client on Linux, Windows, and macOS
 
-Extract and install the `virtctl` client after downloading from the appropriate location for your operating system.
+Download and install the `virtctl` client for your operating system.
 
-.Prerequisites
+.Procedure
 
-* You must have downloaded the `virtctl` client.
+. Navigate to *Virtualization > Overview* in the {product-title} web console.
 
-.Procedure
+. Click the *Download virtctl* link on the upper right corner of the page and download the `virtctl` client for your operating system.
```

Comment:
```suggestion
. Click the *Download virtctl* link, which is on the upper right corner of the page, and download the `virtctl` client for your operating system.
```
IBM Style suggests to provide additional descriptive info when describing a location on a page
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#u